flaml/model.py

```diff
@@ -550,8 +548,8 @@ def tokenizer(self):
                 self._training_args.model_path,
                 use_fast=True,
                 add_prefix_space=True
-                if "roberta" in self._training_args.model_path
-                else False,  # If roberta model, must set add_prefix_space to True to avoid the assertion error at
+                if ("roberta" in self._training_args.model_path and self.fit_flag)

```

Comment:
don't use if...else...
Use the expression directly.

Reply:
Is `tokenizer()` an internal function only used by `fit()` or `predict()`? If so, rename it into `_tokenizer()` and add comment that it can only used by `fit()` or `predict()`.